### PR TITLE
Added a welcome tab

### PR DESCRIPTION
**Pull request by diomidov**
Part of #87

### CODE REVIEW

File: client/Meeting.coffee [hunk 1]
```diff
@@ -154,6 +155,17 @@ export Meeting = ->
...
       Session.set 'currentRoom', undefined
+    
+    if model.getRoot().getChildren().every (tabset) -> tabset.getChildren().length == 0
+      welcomeTabJson =
+        id: 'welcome'
+        type: 'tab'
+        name: 'Welcome'
+        component: 'Welcome'
+        enableClose: true
+        enableDrag: true
```
Comment (edemaine) [inline client/Meeting.coffee:166]: Can't this be added directly to `initModel`? Or is the point to re-add it when someone closes it and everything else? (Not sure whether that's annoying...)  I had in mind that `enableClose` would be `false`...

You also don't need to set these `enable`s explicitly to `true`; they're the default.

File: client/Meeting.coffee [hunk 2]
```diff
@@ -209,7 +225,7 @@ export Meeting = ->
         </div>
     else if node.getComponent() == 'ChatRoom'
       return ChatRoom.onRenderTab node, renderState
-    return if node.getParent().getType() == 'border'
+    return if node.getComponent() != 'Room'
```
Comment (edemaine) [inline client/Meeting.coffee:228]: Nice improvement.

File: client/Welcome.coffee [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+
+export Welcome = () =>
+  <div className='welcome-page'>
```
Comment (edemaine) [inline client/Welcome.coffee:4]: Maybe use `<Card>`, like `TabNew`?  Then shouldn't need any special CSS.

File: client/Meeting.coffee (hunk 1 above)
Comment (diomidov) [reply client/Meeting.coffee:166]: I was thinking an unclosable tab would be even more annoying

Comment (edemaine) [reply client/Meeting.coffee:166]: As discussed: then probably shouldn't magically come back when you leave all rooms. Can always reload Comingle if you really want the welcome tab back.

Comment (edemaine) [reply client/Meeting.coffee:166]: Oops, reloading won't restore it, because of localstorage magic. Maybe it's better to leave this as is...

Comment (edemaine) [reply client/Meeting.coffee:166]: Oops again. We don't store the layout for meetings, only tabs within meetings, so this will actually work.

**Pull request merged**